app/controllers/api/api_controller.rb

```diff
@@ -1,5 +1,13 @@
 module Api
   class ApiController < ApplicationController
     skip_before_action :check_account
+
+    def logged_in
+      if logged_in?
+        return render json: {success: true}
+      else
+        return render json: {success: false, status: 403}
```

Comment:
So, this is just a discussion point. you request was technically successful 200 - you are returning the correct payload etc. Also this url is a query, for a pure RESTafarian- 💩 head - should it be `~/session?exist` kind of thing? again, this is purely surface stuff.